manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store",
-  "version": "2.107.0",
+  "version": "2.108.0-beta.0",
```

Comment:
Don't forget to change this back before merging

Reply:
The publishing robot does this 👍